multiapps-controller-persistence/src/main/java/org/cloudfoundry/multiapps/controller/persistence/model/AccessToken.java

```diff
@@ -0,0 +1,24 @@
+package org.cloudfoundry.multiapps.controller.persistence.model;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.immutables.value.Value;
+
+import java.util.Date;
+
+@Value.Immutable
+@JsonSerialize(as = ImmutableAccessToken.class)
+@JsonDeserialize(as = ImmutableAccessToken.class)
+public interface AccessToken {
+
+    @Value.Default
```

Comment:
why 0 as default value?

Reply:
This way you don't need to set the id in the tests